fashrevwall/run_streaming.py

```diff
@@ -0,0 +1,8 @@
+from TwitterClient import TwitterClient
```

Comment:
the streaming capability should actually be converted into a django management command, then we can run `python manage.py get twitterimages` or smth like that when we need to and can tell heroku to run this every X number of minutes